### PR TITLE
lunr - TokenSet definition filled out

### PR DESCRIPTION
**Pull request by dkershner6**
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://lunrjs.com/docs/lunr.TokenSet.html

### CODE REVIEW

File: types/lunr/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Type definitions for lunr.js 2.3
 // Project: https://github.com/olivernn/lunr.js, http://lunrjs.com
-// Definitions by: Sean Tan <https://github.com/seantanly>, Andrés Pérez <https://github.com/hokiegeek>
+// Definitions by: Sean Tan <https://github.com/seantanly>, Andrés Pérez <https://github.com/hokiegeek>, Derek Kershner <https://github.com/dkershner6>
```
Comment (peterblazejewicz) [inline types/lunr/index.d.ts:3]:
```suggestion
// Definitions by: Sean Tan <https://github.com/seantanly>
//				   Andrés Pérez <https://github.com/hokiegeek>
//				   Derek Kershner <https://github.com/dkershner6>
```

File: types/lunr/index.d.ts [hunk 2]
```diff
@@ -754,15 +757,31 @@ declare namespace lunr {
      * This helps to reduce the space used for storing the token set.
      */
     class TokenSet {
-        constructor()
+        constructor();
+
+        id: number;
```
Comment (peterblazejewicz) [inline types/lunr/index.d.ts:762]:
```suggestion
		/** @private */
        id: number;
```

File: types/lunr/index.d.ts [hunk 3]
```diff
@@ -754,15 +757,31 @@ declare namespace lunr {
...
+        final: boolean;
```
Comment (peterblazejewicz) [inline types/lunr/index.d.ts:763]:
```suggestion
		/** @package */
        final: boolean;
```

File: types/lunr/index.d.ts [hunk 4]
```diff
@@ -754,15 +757,31 @@ declare namespace lunr {
...
+
+        /**
+         * Key is a single character, the letter that represents the
```
Comment (peterblazejewicz) [inline types/lunr/index.d.ts:766]:
```suggestion
		 * @package
         * Key is a single character, the letter that represents the
```

**Comment by elibarzilay** [comment]
@dkershner6 ping re @peterblazejewicz's suggestions.

File: types/lunr/index.d.ts (hunk 4 above)
Comment (andrewbranch) [reply types/lunr/index.d.ts:766]: What are the `@package` comments for?

Comment (dkershner6) [reply types/lunr/index.d.ts:770]: I dunno either, but happy to add them.

**Comment by dkershner6** [comment]
> @dkershner6 ping re @peterblazejewicz's suggestions.

Thank you, @elibarzilay , sorry for the delay.

File: types/lunr/index.d.ts (hunk 4 above)
Comment (andrewbranch) [reply types/lunr/index.d.ts:771]: The whitespace formatting is kind of all over the place now (I think there’s a mix of tabs and spaces), and I want to know what these tags are for—@peterblazejewicz?

**Review by andrewbranch — CHANGES_REQUESTED** [review]

**Comment by peterblazejewicz** [comment]
@andrewbranch those are optional, only to mark those are private apis, not indented to be used when consuming package. These are grayish area, we could skip those (while still making sure the same white-space format is applied to PR)

**Pull request closed without merging**